ironfish-cli/src/commands/service/bridge/deposit.ts

```diff
@@ -96,6 +95,11 @@ export class Deposit extends IronfishCommand {
 
     const to = flags.to ?? (await api.getBridgeAddress())
 
+    let dest = flags.dest
```

Comment:
Looks good if default is null, not sure if flags defaults to null or undefined but I am guessing you tested :)